card/mod/05_standard/set/all/rich_html/menu.rb

```diff
@@ -118,17 +118,17 @@ def show_menu_items
       :structure  => show_structure?,
       :discuss    => disc_card && disc_card.ok?( disc_card.new_card? ? :comment : :read ),
       :page       => !main?,
-      :more       => true
+      :more       => !card.new_card?
     }
     if card.real?
       res.merge!(
-        :edit      => card.ok?(:update) || show_structure?,
+        :edit      => !card.new_card? && (card.ok?(:update) || show_structure?),
```

Comment:
is this necessary?  we're already in "card.real?"  if it's real, it's not new, right?